content/en/blog/releases/v0.6.0.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Build v0.6.0"
+date: 2021-10-11T16:00:00-04:00
+draft: false
+author: "Sascha Schwarze ([@SaschaSchwarze0](https://github.com/SaschaSchwarze0))"
+---
+
+It was a long journey, longer than it should have been, but we have finally reached our goal! Shipwright Build v0.6.0, our first release after joining the [Continuous Delivery Foundation](https://cd.foundation/).
```

Comment:
Our prior release announcements tended to be more terse - mainly pointing to the release notes on GitHub.

This page should have the filename `v0.6.0-release-notes.md`, and highlight the breaking changes/upgrade instructions. For some reason the post is not rendering - are you able to build the site locally with Hugo?

Perhaps move this content to a different file the `posts/` directory, and give it a filename that starts with the blog post date? Ex - `2021-10-11-build-release-v0.6.0`